### PR TITLE
sh-36 feature (build) added materialize css and scss loader

### PR DESCRIPTION
**Pull request by disist**

### CODE REVIEW

File: env/webpack.common.js [hunk 1]
```diff
@@ -1,31 +1,25 @@
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
-
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const envConfig = require('./env.config');
```
Comment (antonSavk) [inline env/webpack.common.js:4]: !fix

Comment (antonSavk) [inline env/webpack.common.js:4]: !fixed

**Comment by kosich** [comment]
!fix @disist please, squash commits into one (theres no need for commit-then-revert guys to be present here), using our git naming convention https://github.com/garage-it/SmartHouse-frontend/wiki/Working-with-git

Also, try avoiding updating branches with merge -- rebase is preferable

**Comment by disist** [comment]
!fixed 
Dear @kosich I will avoiding updating branches with merge, thnks

**Pull request merged**